src/lib/config.ts

```diff
@@ -1366,6 +1369,129 @@ export function removeChangeListener<P extends keyof default_config>(
+                        parseobj.binds.push(`bind --mode=ex ${e} ${pconf[i][e]}`)
+                    } else {
+                        parseobj.binds.push(`unbind --mode=ex ${e}`)
+                    }
+                } else if (i === "ignoremaps" ) {
+                    if (pconf[i][e].length > 0) {
+                        parseobj.binds.push(`bind --mode=ignore ${e} ${pconf[i][e]}`)
+                    } else {
+                        parseobj.binds.push(`unbind --mode=ignore ${e}`)
+                    }
+                } else if (i === "imaps" ) {
+                    if (pconf[i][e].length > 0) {
+                        parseobj.binds.push(`bind --mode=insert ${e} ${pconf[i][e]}`)
+                    } else {
+                        parseobj.binds.push(`unbind --mode=insert ${e}`)
+                    }
+                } else if (i === "inputmaps" ) {
+                    if (pconf[i][e].length > 0) {
+                        parseobj.binds.push(`bind --mode=input ${e} ${pconf[i][e]}`)
+                    } else {
+                        parseobj.binds.push(`unbind --mode=input ${e}`)
+                    }
+                } else if (i === "hintmaps" ) {
+                    if (pconf[i][e].length > 0) {
+                        parseobj.binds.push(`bind --mode=hint ${e} ${pconf[i][e]}`)
+                    } else {
+                        parseobj.binds.push(`unbind --mode=hint ${e}`)
+                    }
+                } else if (i === "subconfigs") {
+                    parseobj.subconfigs.push(`js tri.config.set("${i}", {"${e}": ${JSON.stringify(pconf[i][e])}})`)
+                } else if (i === "exaliases") {
+                    // Only really useful if mapping the entire config and not just pconf.
+                    if (e === "alias") {
+                        parseobj.aliases.push(`command ${e} ${pconf[i][e]}`)
+                    } else {
+                        parseobj.aliases.push(`alias ${e} ${pconf[i][e]}`)
+                    }
+                } else if (i === "autocmds") {
+                    for (const a of Object.keys(pconf[i][e])) {
+                        parseobj.aucmds.push(`autocmd ${e} ${a} ${pconf[i][e][a]}`)
+                    }
+                } else if (i === "autocontain") {
+                    parseobj.aucmds.push(`autocontain ${e} ${pconf[i][e]}`)
+                } else if (i === "logging") {
+                    // Map the int values in e to a log level
+                    let level
+                    if (pconf[i][e] === 0) level = "never"
+                    if (pconf[i][e] === 1) level = "error"
+                    if (pconf[i][e] === 2) level = "warning"
+                    if (pconf[i][e] === 3) level = "info"
+                    if (pconf[i][e] === 4) level = "debug"
+                    parseobj.logging.push(`set logging.${e} ${level}`)
+                } else {
+                    parseobj.conf.push(`set ${i}.${e} ${pconf[i][e]}`)
```

Comment:
Surely this line should be identical to the subconfigs one for future proofing?

Reply:
Yeah you're right. Totally forgot to update it.